src/themes/default/components/core/blocks/Wishlist/Product.vue

```diff
@@ -3,7 +3,6 @@
     <div @click="closeWishlist">
       <router-link :to="localizedRoute({
         name: product.type_id + '-product',
-        fullPath: product.url_path,
```

Comment:
I don't think this is the right solution. By setting `fullPath` we inform `localizedRoute` to use the `localizedDispatcherRoute` that generates the seo-friendly URL. It should consist the `?childSku` parameter anyway. It's done by this line: https://github.com/DivanteLtd/vue-storefront/blob/20b693e28eeac810252593e1b42e709cc03701e9/core/lib/multistore.ts#L137

If it's not there (the childSku) we need to find the reason